docs/utils/mode.ts

```diff
@@ -0,0 +1,10 @@
+import {useRequestContext} from '@blinkk/root';
+
+export function testPreviewMode() {
+  return !!testQueryParam('preview');
+}
+
+export function testQueryParam(param: string) {
+  const ctx = useRequestContext();
+  return ctx.props.req?.query?.[param];
```

Comment:
test() fns should generally return a boolean value (afaik this return `undefined | string | string[]`

Reply:
done